pkg/sql/schemachanger/scbuild/builder_state.go

```diff
@@ -364,7 +369,7 @@ func (b *builderState) NextTableConstraintID(id catid.DescID) (ret catid.Constra
 		}
 	}
 
-	b.QueryByID(id).ForEachElementStatus(func(_ scpb.Status, _ scpb.TargetStatus, e scpb.Element) {
+	b.QueryByID(id).ForEach(func(_ scpb.Status, _ scpb.TargetStatus, e scpb.Element) {
```

Comment:
Arguably we could have called `ForEachElement` instead. To be done at a later date.